src/bundle/Resources/public/scss/_inputs.scss

```diff
@@ -85,6 +89,13 @@
         }
     }
 
+    &--textarea {
+        transition: all $ibexa-admin-transition-duration $ibexa-admin-transition, box-shadow 0s; // Chrome does not render box-shadow properly with transition
```

Comment:
Note for future with image showing mentioned rendering problems:
![Screenshot 2021-07-13 at 10 40 11](https://user-images.githubusercontent.com/10233057/128341504-d86060b8-930d-43e0-96a3-f4621748626b.png)